app/admin/legislations.rb

```diff
@@ -27,6 +27,38 @@
     actions
   end
 
+  show do
+    tabs do
+      tab :details do
+        attributes_table do
+          row :title
+          row :description
+          row :date_passed
+          row :location
+          row :law_id
+          row :framework
+          row :created_at
+          row :updated_at
+        end
+      end
+
+      tab :litigations do
```

Comment:
We have to think about standardizing this across the Panel. Sometimes we have it in a form of a list within a row, and sometimes as separate tabs. We can do that later when we have most of the entities in place and we will see what makes more sense.